lib/features/digital_guide_view/general_info/presentation/widgets/report_change_button.dart

```diff
@@ -35,3 +41,23 @@ class ReportChangeButton extends StatelessWidget {
     );
   }
 }
+
+Future<void> openEmailApp(BuildContext context, WidgetRef ref) async {
+  final errorMessageToast = context.localize.report_change_error_toast_message;
+  final backgroundColorToast = context.colorTheme.greyLight;
+  final textColorToast = context.colorTheme.blackMirage;
+
+  if (await ref.launch(
+    "mailto:${context.localize.report_change_email}?subject=${Uri.encodeComponent(context.localize.report_change_subject)}",
+  )) {
+    unawaited(
+      Fluttertoast.showToast(
+        msg: errorMessageToast,
+        toastLength: Toast.LENGTH_LONG,
+        gravity: ToastGravity.BOTTOM,
+        backgroundColor: backgroundColorToast,
+        textColor: textColorToast,
+      ),
+    );
+  }
+}
```

Comment:
this function could be private. Imo whole content could be moved to lambda call. It's not that long